dbf2csv/wizard/dbf2csv.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <record id="dbf_file_process_view" model="ir.ui.view">
+        <field name="name">Conversions</field>
+        <field name="model">dbftocsv.conversion</field>
+        <field name="arch" type="xml">
+            <form string="Select DBF file">
+                <sheet>
+                    <h2>Upload a DBF file and convert to CSV</h2>
+                    <group>
+                        <group string="DBF">
+                            <field name="data_file" filename="filename" placeholder="Choose a DBF file ..."/>
+                            <field name="filename" invisible="1"/>
+                        </group>
+                        <group string="CSV">
+                            <field name="data_file_csv" filename="filename_csv" readonly="1"/>
+                            <field name="filename_csv" invisible="1"/>
+                        </group>
+                    </group>
+                    <footer>
+                        <button type="object"
+                            name="process_file"
+                            string="Convert to csv"
```

Comment:
Nitpick: I think _CSV_ should be uppercased.

Reply:
Ok, :memo: